.github/workflows/pull_request.yaml

```diff
@@ -354,3 +354,4 @@ jobs:
           format: 'table'
           trivyignores: '.github/workflows/.trivyignore'
           exit-code: 1
+          skip-files: '/app/docs/configure/bitbucket.html'
```

Comment:
@jgadsden - I did _technically_ find a way to get it to skip the file in question.  

I'm not a huge fan of skipping the file entirely, or the potential of the list of skipped files growing in the future.  That said, it is a static file for our documentation, so I feel that the risk is low enough and we can keep getting the green light from our Trivy scan.  Let me know if you're on board with this or if you'd like me to continue exploring alternatives. :)

Reply:
I agree with your solution - we should skip the docs files
I have had feedback in the past that having the env vars spelt out has been very useful, and that is what Trivy is complaining about (or at least the ones that say secrets)
let's go with this solution and merge

after a two year gap, we are back to nicely formatted docs 🥇